synapse/lib/stormtypes.py

```diff
@@ -546,16 +555,19 @@ class LibService(Lib):
          'type': {'type': 'function', '_funcname': '_libSvcGet',
                   'args': (
                       {'name': 'name', 'type': 'str',
-                       'desc': 'The local name, local iden, or remote name, of the service to get the definition for.', },
+                       'desc':
+                           'The local name, local iden, or remote name, of the service to get the definition for.', },
```

Comment:
This seems inconsistent with the desc approach below to ensure content fits width restrictions.  Since the convention below can extend beyond saving 4 chars, lets go with that.